Sources/Apodini/Semantic Model Builder/OpenAPI/Extensions/Operation+OpenAPI.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Created by Lorena Schlesinger on 15.11.20.
+//
+
+import Foundation
+@_implementationOnly import OpenAPIKit
+
+/// Extension to map Apodini `Operation`  to `OpenAPI.HttpMethod`.
+extension Operation {
```

Comment:
I'd follow the same approach here as well (https://github.com/Apodini/Apodini/pull/35#discussion_r552936299)

```suggestion
extension OpenAPI.HttpMethod {
    init(_ operation: Operation) {
        switch operation {
        case .automatic, .read:
            self = .get
        case .create:
            self = .post
        case .update:
            self = .put
        case .delete:
            self = .delete
        }
    }
}
```

Reply:
solved by 5b47895